server/api.go

```diff
@@ -50,12 +50,12 @@ type dexAPI struct {
 }
 
 type vehicleIDTokenClaims struct {
-	Issuer   string   `json:"iss"`
-	Subject  string   `json:"sub"`
-	Audience audience `json:"aud,omitempty"`
-	Expiry   int64    `json:"exp"`
-	IssuedAt int64    `json:"iat"`
-	UserID   string   `json:"userId"`
+	Issuer         string   `json:"iss"`
+	Subject        string   `json:"sub"`
+	Audience       audience `json:"aud,omitempty"`
+	Expiry         int64    `json:"exp"`
+	IssuedAt       int64    `json:"iat"`
+	UserEthAddress string   `json:"userId"`
```

Comment:
```suggestion
	UserEthAddress string   `json:"userEthAddress"`
```
This is the only serious one.

Reply:
Another thing to say is that it seems somewhat well established that outside of the three-character fields like `nbf` the names in JWTs are `snake_cased`, see [the registry](https://www.iana.org/assignments/jwt/jwt.xhtml). Up to you whether this is important or not.